flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/failover/flip1/RestartPipelinedRegionFailoverStrategy.java

```diff
@@ -84,34 +77,8 @@ public RestartPipelinedRegionFailoverStrategy(
 		ResultPartitionAvailabilityChecker resultPartitionAvailabilityChecker) {
 
 		this.topology = checkNotNull(topology);
-		this.regions = Collections.newSetFromMap(new IdentityHashMap<>());
-		this.vertexToRegionMap = new HashMap<>();
 		this.resultPartitionAvailabilityChecker = new RegionFailoverResultPartitionAvailabilityChecker(
 			resultPartitionAvailabilityChecker);
-
-		// build regions based on the given topology
-		LOG.info("Start building failover regions.");
-		buildFailoverRegions();
-	}
-	// ------------------------------------------------------------------------
-	//  region building
-	// ------------------------------------------------------------------------
-
-	private void buildFailoverRegions() {
-		final Set<Set<SchedulingExecutionVertex>> distinctRegions =
-			PipelinedRegionComputeUtil.computePipelinedRegions(topology);
-
-		// creating all the failover regions and register them
-		for (Set<SchedulingExecutionVertex> regionVertices : distinctRegions) {
-			LOG.debug("Creating a failover region with {} vertices.", regionVertices.size());
-			final FailoverRegion failoverRegion = new FailoverRegion(regionVertices);
-			regions.add(failoverRegion);
-			for (SchedulingExecutionVertex vertex : regionVertices) {
-				vertexToRegionMap.put(vertex.getId(), failoverRegion);
-			}
-		}
-
-		LOG.info("Created {} failover regions.", regions.size());
```

Comment:
How about adding such logs in DefaultExecutionTopology region building?
This can be helpful to diagnose whether regions are built as expected. 
And maybe also one line for region building start so that users can see how much time the region building takes.

Reply:
Makes sense.

Reply:
Added logging (153983502e03705fa2496728c1ef76167a6fe68c). Wasn't sure about the previous log level. It's on INFO level now.

Reply:
I'd prefer to make it INFO. Regions are basic components now but users can get little information on that.
And it is a short and one time log so it would not mess up the logs.

Reply:
Ok makes sense. Changed to info.